### DISCUSSION TITLE
Need a short video

### DISCUSSION THREAD

**Discussion by uuc110** [code-kern-ai/bricks#170] [Ideas]
I guess it would be better if there is a small clip to see a demo of repo what it do

**Comment by jhoetter** [comment]
Comes tomorrow :)

**Reply by jhoetter** [reply]
that went faster than i expected. it's already created and now part of the Readme @uuc110 :)
you can find it here: https://www.youtube.com/watch?v=Wcbdwwr5AI8&ab_channel=KernAI
